Tests/SonarQube.TeamBuild.PreProcessor.Tests/DataModel/QualityProfile.cs

```diff
@@ -37,10 +40,23 @@ public QualityProfile AddRule(Rule rule)
             return this;
         }
 
+        public void SetExport(string format, string content)
+        {
+            this.formatToContentExportMap[format] = content;
+        }
+
         public string Name { get { return this.name; } }
         public string Language { get { return this.language; } }
         public IEnumerable<string> Projects { get { return this.projectKeys; } }
-        public IEnumerable<Rule> ActiveRules { get { return this.activeRules; } }
-        
+        public ISet<Rule> ActiveRules { get { return this.activeRules; } }
```

Comment:
I changed the type to ISet<> so that tests can clear the set of active rules if desired.